draftivist_api/api/serializers.py

```diff
@@ -36,12 +38,17 @@ class Meta:
 
 class CampaignSerializer(serializers.ModelSerializer):
     recipients = RecipientSerializer(many=True, read_only=True)
-    issues = IssueSerializer(many=True, read_only=True)
+    issues = serializers.SerializerMethodField()
     group = GroupSerializer(read_only=True)
 
     class Meta:
         model = Campaign
-        fields = ['id', 'name', 'description', 'group', 'start_date', 'end_date', 'created', 'recipients', 'issues', 'is_active', 'allow_custom_statements']
+        fields = ['id', 'name', 'description', 'group', 'start_date', 'end_date', 'recipients', 'issues',
```

Comment:
is `recipients` who the emails will be sent to?

Reply:
Shouldn't we keep `allow_custom_statements`?

Reply:
Yep. Local DB fixtures will be updated in a followup PR to pre-populate this for the sample campaign.